frontend/src/app/App.test.tsx

```diff
@@ -228,9 +228,7 @@ const MODAL_TEXT = "Welcome to the SimpleReport";
 
 describe("App", () => {
   beforeEach(() => {
-    jest
-      .useFakeTimers("modern")
-      .setSystemTime(new Date("2021-08-01").getTime());
+    jest.useFakeTimers().setSystemTime(new Date("2021-08-01").getTime());
```

Comment:
In the latest version of jest the ability to set the type of timer was removed and "modern" is now the standard. https://jestjs.io/blog/2020/05/05/jest-26